recipes/censo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "censo" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/grimme-lab/censo/archive/refs/tags/v.{{ version }}.tar.gz
+  sha256: f1d77d8eb9d25fe4157491a5d298321f84999123970411831b059cea39aecad2
+
+build:
+  number: 0
+  entry_points:
+    - {{ name }} = censo_qm.censo:main
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
```

Comment:
There is a list of requirements here: https://github.com/grimme-lab/censo#requirements

Are they not needed?

Reply:
Added a run requirement on `xtb`.

While Orca is free for academic use, I can't package it here, however the program will search for a user installation and pick it up if available in the `PATH`. I hope that is okay.

The Turbomole, COSMOtherm, cefine dependencies require a commercial license, but is not required for the basic functionality.

Reply:
Ok that's fine for me. Does it make sense to add them under `run_constrained` or are they not suited for conda (Not necessarily conda-forge) at all?